persistent/test/Database/Persist/TH/ForeignRefSpec.hs

```diff
@@ -102,6 +102,15 @@ spec = describe "ForeignRefSpec" $ do
     it "should compile" $ do
         True `shouldBe` True
 
+    describe "ForeignSchemaName" $ do
+        let
+            [childForeignDef] =
+                entityForeigns $ entityDef $ Proxy @ChildExplicit
+        it "should have a schema name defined" $ do
+            (foreignRefSchemaDBName childForeignDef)
```

Comment:
I'd prefer to use `NamedFieldPuns` here but adding a pragma for a couple of lines seems excessive.